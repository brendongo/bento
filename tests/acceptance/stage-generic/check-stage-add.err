Running Bento checks on staged files...

⚠ 1 finding(s) due to staged changes in  s

Please fix these issues, or:

◦ To archive findings as tech debt, run $ bento archive
◦ To disable a specific check, run $ bento disable check TOOL CHECK
