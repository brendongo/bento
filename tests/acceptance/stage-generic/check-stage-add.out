flake8 undefined-name https://lintlyci.github.io/Flake8Rules/rules/F821.html
     > bar.py:1
     ╷
    1│   y = z
     ╵
     = undefined name 'z'

